packages/editor-ui/src/views/NodeView.v2.vue

```diff
@@ -832,6 +830,66 @@ function registerCustomActions() {
 	// });
 }
 
+/**
+ * Routing
+ */
+
+onBeforeRouteLeave(async (to, from, next) => {
```

Comment:
Since we have this logic here, we could get rid of the `promptSaveOnBeforeRouteLeave` method, which isn't used, right?

Reply:
Done! Forgot about it. Thanks!